src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Folding/FoldingRangeEndpoint.cs

```diff
@@ -186,7 +178,9 @@ private FoldingRange FixFoldingRangeStart(FoldingRange range, RazorCodeDocument
             // +1 to the offset value because the helper goes to the character position
             // that we want to be after. Make sure we don't exceed the line end
             var newCharacter = Math.Min(offset.Value + 1, lineSpan.Length);
+
             range.StartCharacter = newCharacter;
+            range.CollapsedText = "...";
```

Comment:
Should we set this to null/empty so that the editor chooses what to display, rather than forcing "..."? Seems like this should probably be a client thing, and that just moving the start character should be enough to get it to do the old behaviour.

Reply:
Let me see what that looks like...

Reply:
works for me :shipit: